ibis/config.py

```diff
@@ -97,6 +97,7 @@ class Interactive(Config):
     max_string: int = 80
     max_depth: int = 1
     show_types: bool = True
+    console_width: Optional[int] = None
```

Comment:
I don't think exposing this option to the user makes much sense. I'd rather figure out what we need to do to make the doctests reproducible without changing anything user facing.

Reply:
I agree.

Reply:
We can use another global variable to force a terminal width when running the doctests, though I think exposing it could be useful too, perhaps with a different name. 

This option is similar to `max_columns` but basically forces the table repr to fit in a certain number of characters.

Reply:
Actually, nevermind, that would set it for all users and force it to be static.

Reply:
I pushed up an alternative fix at #5999 that I believe should resolve this issue without the need for a new option.